AnnoyancesFilter/Widgets/sections/widgets.txt

```diff
@@ -2261,7 +2261,9 @@ samlab.ws##iframe[src^="https://yoomoney.ru/"]
 !
 hromadske.ua#$#.DonateModal-overlay { display: none !important; }
 hromadske.ua#$#body { overflow: visible !important; }
-!
+inkorr.com##.sub-tg-wrapper
+inkorr.com##.views-row:has(.fa-telegram)
+inkorr.com##.tg-box
```

Comment:
Looks like an ad. Add it to the `CyrillicFilters - common-sections - specific.txt`.